virtcontainers/physical_endpoint.go

```diff
@@ -98,9 +100,31 @@ func (endpoint *PhysicalEndpoint) Detach(netNsCreated bool, netNsPath string) er
 	return bindNICToHost(endpoint)
 }
 
-// HotAttach for physical endpoint not supported yet
+// HotAttach for physical endpoint
 func (endpoint *PhysicalEndpoint) HotAttach(h hypervisor) error {
-	return fmt.Errorf("PhysicalEndpoint does not support Hot attach")
+	networkLogger().WithField("endpoint-type", "physical").Info("Attaching endpoint (hotplug)")
```

Comment:
Suggestion:

```suggestion
	networkLogger().WithField("endpoint-type", "physical").Info("Hot attaching endpoint")
```

Alternatively, add a field for `hotplug`.